torch/_inductor/cudagraph_trees.py

```diff
@@ -955,6 +955,9 @@ def _copy_inputs_and_remove_from_src(self, dsts, srcs):
         # Fails on empty lists
         if dst_tensors:
             torch._foreach_copy_(dst_tensors, src_tensors)
+            counters["inductor"]["cudagraph_copies_due_to_non_static_inputs"] += len(
+                dst_tensors
+            )
 
```

Comment:
We should not be updating counters on the hot path. can you set in this in CudaGraphNode `__init__` ?

Reply:
moving this into __init__ needs to change the semantics to "cudagraph_recorded_non_static_inputs"

Reply:
Just use len(`non_static_input_idx`)

Reply:
yeah i'll use that, it just won't be as informative, since recordings can be thrown away, and it no longer measures how many times the copies happen